readthedocs.yml

```diff
@@ -2,7 +2,7 @@ build:
   image: latest
 
 python:
-  version: 3.6
+  version: 3.8
```

Comment:
is this necessary for anything else? Everything in the docs should become 3.6 compatible with this PR.
Maybe we can leave this in to see if RTD honours this.

Reply:
Works perfect with 3.6.